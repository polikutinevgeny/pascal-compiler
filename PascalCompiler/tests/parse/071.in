const
   h = 20;
var
   a: integer;
   b: array[1..2] of integer;
begin
   write(a + 2, h mod 3, b[2]);
end.