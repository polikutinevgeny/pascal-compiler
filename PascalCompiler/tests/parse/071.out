└─Program ''
  └─Block
    ├─Declarations
    │ ├─Constant section
    │ │ └─h
    │ │   └─20
    │ └─Var section
    │   ├─Variable declaration
    │   │ ├─integer
    │   │ └─a
    │   └─Variable declaration
    │     ├─Array
    │     │ ├─integer
    │     │ └─Range
    │     │   ├─1
    │     │   └─2
    │     └─b
    └─Compound statement
      └─Statement list
        └─Write
          └─Expression list
            ├─+
            │ ├─Designator
            │ │ └─a
            │ └─2
            ├─mod
            │ ├─Designator
            │ │ └─h
            │ └─3
            └─Designator
              ├─b
              └─Expression list
                └─2
